nisystemlink/clients/testmonitor/models/_api_info.py

```diff
@@ -0,0 +1,72 @@
+from nisystemlink.clients.core._api_info import Operation
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+
+
+class V2Operations(JsonModel):
+    """The operations available in the routes provided by the v2 HTTP API."""
+
+    get_products: Operation
+    """The ability to get a list of products."""
+
+    query_products: Operation
+    """The ability to query Products based on their properties."""
```

Comment:
nit: We capitalized Products here but not in other places

Reply:
good catch. Fixed that one.